apps/voting/contracts/Voting.sol

```diff
@@ -290,30 +292,44 @@ contract Voting is IForwarder, AragonApp {
     */
     function vote(uint256 _voteId, bool _supports, bool _executesIfDecided_deprecated) external voteExists(_voteId) {
         require(_canVote(_voteId, msg.sender), ERROR_CAN_NOT_VOTE);
-        require(!_supports || _getVotePhase(votes[_voteId]) == VotePhase.Main, ERROR_CAN_NOT_VOTE);
+        require(_canCastYeaVote(_voteId, _supports), ERROR_CAN_NOT_VOTE);
```

Comment:
This function is a bit confusing. It returns true, meaning that the voter can cast a yea vote when he actually votes against it, passing `_supports` equal to false. A bit illogical